components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/impl/APIUsageStatisticsRestClientImpl.java

```diff
@@ -2255,6 +2255,12 @@ public int compare(APIUsageDTO o1, APIUsageDTO o2) {
     @Override
     public void deployArtifacts(String url, String user, String pass) throws Exception {
 
+        if (url.trim().equals("")) {
```

Comment:
removed in commit: https://github.com/wso2/carbon-apimgt/commit/9619da2271b49f37ef93456b6f8a8ac235f685ea because empty URL is not valid in this scenario.